src/components/control/Chip/Chip.scss

```diff
@@ -0,0 +1,130 @@
+evg-chip {
+  --_color: var(--evg-chip-color-light);
+  --_background: var(--evg-chip-background-light);
+  --_background-hover: var(--evg-chip-background-hover-light);
+  --_border-color: var(--evg-chip-border-color-light);
+  --_border-color-hover: var(--evg-chip-border-color-hover-light);
+
+  --_color-selected: var(--evg-chip-color-selected-light);
+  --_background-selected: var(--evg-chip-background-selected-light);
+  --_background-hover-selected: var(--evg-chip-background-hover-selected-light);
+  --_border-color-selected: var(--evg-chip-border-color-selected-light);
+  --_border-color-hover-selected: var(
+    --evg-chip-border-color-hover-selected-light
+  );
+
+  &[variant='dark'] {
```

Comment:
Can the variant go at the  bottom, always try to keep stuff in C -> E -> A order as it helps the cascade, although I can see why it's up here